cfme/tests/containers/test_smart_management.py

```diff
@@ -42,30 +43,35 @@ def get_object_name(obj):
 
 
 def set_random_tag(instance):
+    logger.info("Setting random tag")
     navigate_to(instance, 'Details')
     toolbar.select('Policy', 'Edit Tags')
 
     # select random tag category
     cat_selector = AngularSelect("tag_cat")
     random_cat = random.choice(cat_selector.all_options)
+    logger.info("Selected category {cat}".format(cat=random_cat))
     cat_selector.select_by_value(random_cat.value)
 
     # select random tag tag
     tag_selector = AngularSelect("tag_add")
     random_tag = random.choice([op for op in tag_selector.all_options if op.value != "select"])
+    logger.info("Selected value {tag}".format(tag=random_tag))
     tag_selector.select_by_value(random_tag.value)
 
     # Save tag conig
     form_buttons.save()
-
+    logger.info("Tag configuration was saved")
     return Tag(display_name=random_tag.text, category=random_cat.text)
 
 
 def wait_for_tag(obj_inst):
     # Waiting for some tag to appear at "My Company Tags" and return pop'ed last tag
-    return wait_for(lambda: getattr(obj_inst.summary.smart_management,
+    last_tag = wait_for(lambda: getattr(obj_inst.summary.smart_management,
```

Comment:
**required** please shift the indentation around so the details align again

Reply:
what do you mean by "shift the indentation"?

Reply:
sometihng like 
```python
last_tag = wait_for(
    lambda: getattr(obj_inst.summary.smart_management,'my_company_tags', []),
    fail_condition=[],
    fail_func=obj_inst.summary.reload,
    num_sec=30, delay=5,
).out